composer.json

```diff
@@ -27,24 +27,27 @@
         "test": "bin/asynit tests/"
     },
     "require": {
-        "php": "^7.3 || ^8.0",
-        "amphp/http-client": "^3.0.14",
-        "amphp/sync": "^1.0",
+        "php": "^8.1",
+        "amphp/http-client-psr7": "v2.x-dev",
         "bovigo/assert": "^6.2",
-        "doctrine/annotations": "^1.2",
         "doctrine/collections": "^1.3",
-        "guzzlehttp/psr7": "^1.4",
-        "php-http/client-common": "^2.0",
-        "php-http/message": "^1.3",
+        "guzzlehttp/psr7": "^2.1.0",
         "symfony/console": "^4.4 || ^5.0",
         "symfony/dom-crawler": "^4.4 || ^5.0",
         "symfony/finder": "^4.4 || ^5.0",
         "symfony/yaml": "^4.4 || ^5.0"
```

Comment:
Since you bump to PHP 8.1, you can bump to SF 6.0 (and remove the conflict rule)